src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -203,14 +246,13 @@ def __init__(
             self.trainer.set_seed(seed)
 
         self.a2g = AtomsToGraphs(
-            max_neigh=max_neighbors,
-            radius=cutoff,
             r_energy=False,
             r_forces=False,
             r_distances=False,
-            r_edges=False,
-            r_pbc=True,
+            r_pbc=self.trainer.model.use_pbc,
+            r_edges=self.trainer.model.otf_graph,  # otf graph should not be a property of the model...
```

Comment:
`not self.trainer.model.otf_graph`? We don't want a2g to construct edges since the model will take care of it